CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## 0.9.0 (December 9, 2020)
+
+This release brings to you:
+
+* [#458](https://github.com/redhat-developer/vscode-tekton/issues/458) Update Tkn version to latest.
+* [#448](https://github.com/redhat-developer/vscode-tekton/issues/448) Support Tekton 0.18
+* [#344](https://github.com/redhat-developer/vscode-tekton/issues/344) Validate support for embedded specs
+* [#413](https://github.com/redhat-developer/vscode-tekton/issues/413) Display param descriptions in start pipeline and add trigger
+* [#429](https://github.com/redhat-developer/vscode-tekton/issues/429) Provide completion for Variable Substitutions
```

Comment:
I think you forgot to mention https://github.com/redhat-developer/vscode-tekton/pull/471 issue.